### PR TITLE
Small updates to the docs that update them to latest commands

### PR DESCRIPTION
**Pull request by davekaj**
Small updates! Easy Merge :).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ This will also spin up a GraphiQL interface at `http://127.0.0.1:8000/`.
 6.  Back in the terminal of the subgraph directory, run
 
 ```
+yarn create-subgraph
```
[the remark this answers is not in the document]
Comment (davekaj) [reply README.md]: I added some test in above and below this to make it more clear

**Comment by davekaj** [comment]
all comments addressed

File: docs/getting-started.md [hunk 2]
```diff
@@ -404,7 +407,8 @@ If you want to sync using a public testnet such as Kovan, Rinkeby, or Ropsten, j
...
 
-Now that the infrastructure is set up, you can run `yarn deploy --verbosity debug --watch` in the subgraph directory. `yarn deploy` should have been added to `package.json` in section 1.3 when we took a moment to go through the set up for [Graph CLI documentation](https://github.com/graphprotocol/graph-cli). This builds the subgraph and creates the WASM files in the `/dist` folder. Next, it uploads the `/dist` files to IPFS and deploys it to the Graph Node. The subgraph is now fully running.
+Now that the infrastructure is set up, you can run `yarn create-subgraph` and then `yarn deploy --debug` in the subgraph directory. These commands should have been added to `package.json` in section 1.3 when we took a moment to go through the set up for [Graph CLI documentation](https://github.com/graphprotocol/graph-cli). This builds the subgraph and creates the WASM files in the `dist/` folder. Next, it uploads the `dist/
```
Comment (Jannis) [inline docs/getting-started.md]: Drop the `--debug` here. `yarn deploy` typically includes `--debug`.

**Comment by davekaj** [comment]
adddressed, merging

**Pull request merged**